squad/core/notification.py

```diff
@@ -113,8 +113,11 @@ def __send_notification__(project, notification):
         context=context,
     )
     sender = "%s <%s>" % (settings.SITE_NAME, settings.EMAIL_FROM)
-    for r in recipients:
-        message = EmailMultiAlternatives(subject, text_message, sender, [r.email])
-        if r.html:
-            message.attach_alternative(html_message, "text/html")
-        message.send()
+
+    emails = [r.email for r in recipients]
+    use_html = all([r.html for r in recipients])
```

Comment:
OK so basically a given project is either html emails or it is not. I think this is because you want everyone on the same to: line, but why not put eveyone on reply-to and then send separate emails as before?